etc/inc/rtv-addons.inc

```diff
@@ -0,0 +1,23 @@
+# This file is overwritten during software install.
```

Comment:
In my opinion this should not be an `.inc` file. As `.inc` files tend to be more generic and useful for many more profiles.
Why not rename it `rtv-addons.profile`?
@rusty-snake @glitsj16 What do you think about that?

Reply:
i was copying the firefox include precedent

Reply:
Yes, there are a few exceptions currently (chromium, firefox, feh-network), which I don't really like (inc files also land in the main firejail package on Debian, which I try to keep minimal).
But maybe @rusty-snake or @glitsj16 have a different opinion on that.

Reply:
maybe the main package of firejail should take the generic ones, and the rest come with the profiles? In general the organization of the profiles is one giant mess...

Reply:
> Why not rename it rtv-addons.profile?

@reinerh I agree that would be a much better name. The same goes for the other exceptions mentioned above. Ever since I started to use firejail I added a few mime-types with distinctive icons to visually distinguish between .inc, .local, .net and .profile files. The chromium, firefox & feh-network .inc files indeed stick out as sore thumbs.

Reply:
@pirate486743186 If you need help renaming the file, there's [documentation](https://github.com/blog/1436-moving-and-renaming-files-on-github) on how to do that.